<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormContextDialogBase</class>
 <widget class="QDialog" name="FormContextDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>What do you want to do?</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="pushButtonLinkBarcode">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Link to a barcode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonCaptureData">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Capture Data</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonTakePicture">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Take Picture</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonDeleteData">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Delete Data</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
